### PR TITLE
Poe Objects

### PR DESCRIPTION
**Pull request by Thar0**
`object_poh`, `object_po_composer`, `object_po_field`, `object_po_sisters`, `object_tk`, `object_ps` all `OK`

~~Only unaccounted is 384 bytes at 0x5540 in `object_tk`, I cannot figure out what it could possibly be.~~ (resolved thanks to petrie)

### CODE REVIEW

**Comment by fig02** [comment]
I believe for outname we are omitting the `_tex` from the name, as its only really needed for identifying what it is in the code. Everything else looks good tho

**Review by fig02 — APPROVED** [review]

File: src/overlays/actors/ovl_Bg_Po_Event/z_bg_po_event.c [hunk 1]
```diff
@@ -599,7 +593,7 @@ void BgPoEvent_Update(Actor* thisx, GlobalContext* globalCtx) {
 }
 
 void BgPoEvent_Draw(Actor* thisx, GlobalContext* globalCtx) {
-    static Gfx* displayLists[] = { D_060075A0, D_060079E0, D_06006830, D_06006D60, D_06007230 };
+    static Gfx* displayLists[] = { gPoSistersAmyBlockDL, gPoSistersAmyBethBlockDL, gPoSistersJoellePaintingDL, gPoSistersBethPaintingDL, gPoSistersAmyPaintingDL };
```
Comment (Roman971) [inline src/overlays/actors/ovl_Bg_Po_Event/z_bg_po_event.c:596]: This looks like it needs to be formatted (and probably needs a trailing comma)

File: src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c [hunk 2]
```diff
@@ -151,9 +152,9 @@ static Vec3s D_80ADD7A4[4] = {
 
 static Vec3f D_80ADD7BC = { 120.0f, 250.0f, -1420.0f };
 
-static Gfx* D_80ADD7C8[4] = { 0x06001DE0, 0x06002F88, 0x06003628, 0x06003DC8 };
+static Gfx* D_80ADD7C8[4] = { gPoeSistersMegBodyDL, gPoeSistersJoelleBodyDL, gPoeSistersBethBodyDL, gPoeSistersAmyBodyDL }; // body gfx
 
-static Gfx* D_80ADD7D8[4] = { 0x06001CB0, 0x06002EB8, 0x06003880, 0x06004020 };
+static Gfx* D_80ADD7D8[4] = { gPoeSistersMegFaceDL, gPoeSistersJoelleFaceDL, gPoeSistersBethFaceDL, gPoSistersAmyFaceDL }; // face gfx
```
Comment (Roman971) [inline src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c:157]: same here

**Comment by fig02** [comment]
done

**Review by Roman971 — APPROVED** [review]

**Pull request merged**